assets/diagram-js.css

```diff
@@ -121,8 +121,13 @@
   stroke: var(--element-selected-outline-stroke-color);
 }
 
+.djs-connection.selected .djs-outline {
+  visibility: hidden;
```

Comment:
Why not `display: none`? Do we rely on the outline for hover interactions?

Reply:
I was following our approach for not displaying the outlines: https://github.com/bpmn-io/diagram-js/pull/826/files/af76b072960f45fe0d4d3dc980ab44a6a0495fc1#diff-682322eea3bdf2a0256e2e37fb5a12ac5c9744f340488edcb053b4dbe8f57137R110-R112. But it works with `display` too

Reply:
`visibility: hidden` means usually that we still want the element to affect the layout. However, since it's an SVG element within SVG, it doesn't really affect layout of anything. So it behaves exactly like `display: none` 🤷 .
https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

Reply:
Affect layout and interaction (i.e. for mouse over). The user interaction may be more important here, for (invisible) click interaction boxes.

Reply:
looks like something to be cleaned up some time 🙃

Reply:
TBH the whole discussion we had here suggests `display: none` is a better choice. That wouldn't make me think while `visibility` does.

Reply:
Ok then let's go for the most understandable solution. I updated to use `display`